src/app/frontend/common/components/resourcelist/horizontalpodautoscaler/template.html

```diff
@@ -0,0 +1,30 @@
+<!--
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
```

Comment:
Do you have access to HPAs in this file? Do they load from API?

Check by adding `console.log(x)` in the component or `{{x | json }}` in the template.